PowerShell/BuildingBlocks/CustomerAndResources/AssignSubscriptions.ps1

```diff
@@ -1,23 +1,33 @@
-﻿# Initialize the context
-.\Init.ps1 -Customer
+﻿# Authenticate
```

Comment:
Consider adding a short comment about what the script does here and in other scripts, similar to ManageExtendedProperties.ps1.

Reply:
Done